stylesheet.css

```diff
@@ -0,0 +1,42 @@
+body {
+	background-color: #d3e0e4;
+	font-family: "Trebuchet MS", Helvetica;
```

Comment:
Yay fallbacks. But let's also end any font-family declaration with either `serif` or `sans-serif`. That will mean that even if the browser has none of the fonts we ask for, it'll still fallback to a font of the preferred style.